examples/tabular/example_tabular_predictor_v2.py

```diff
@@ -0,0 +1,62 @@
+
+from autogluon.tabular import TabularPrediction as task
+from autogluon.tabular.task.tabular_prediction.predictor import TabularPredictor
+from autogluon.tabular.task.tabular_prediction.predictor_v2 import TabularPredictorV2
+
+
+################
+# Loading data #
+################
+
+train_data = task.Dataset(file_path='https://autogluon.s3.amazonaws.com/datasets/Inc/train.csv')
+test_data = task.Dataset(file_path='https://autogluon.s3.amazonaws.com/datasets/Inc/test.csv')
+label = 'class'
+hyperparameters = {'RF': {}}
```

Comment:
shall we call it `RandomForest` to avoid confusion, given only a small set of users will construct  `hyperparameters`, making it longer but clearer is ok

Reply:
Will plan to do this (and retain option for 'RF' as additional alias) when we rework the model names (ref #782)